app/static/js/request/new-request-user.js

```diff
@@ -45,6 +45,7 @@ $(document).ready(function () {
 
     $('#request-form').parsley().subscribe('parsley:form:validate', function () {
         // Do stuff when parsley validates
+        // TODO: this or combine (see the other new-request-* js files)
```

Comment:
This is probably important. It's empty for `new-request-user.js` but not for `new-request-anon.js` and `new-request-agency.js`.